src/main/java/bio/terra/app/usermetrics/UserMetricsInterceptor.java

```diff
@@ -69,8 +69,7 @@ public void afterCompletion(
                 BardEventProperties.PATH_FIELD_NAME, path));
     addToPropertiesIfPresentInHeader(
         request, properties, "X-Transaction-Id", BardEventProperties.TRANSACTION_ID_FIELD_NAME);
-    eventProperties.setAll(properties);
-    HashMap<String, Object> bardEventProperties = eventProperties.get();
+    properties.putAll(userLoggingMetrics.get());
```

Comment:
Unless the order of put is important here, this code could instead start with the user logging metrics properties and then add the rest:
```
    Map<String, Object> properties = new HashMap<>(userLoggingMetrics.get());
    properties.putAll(
            Map.of(
                BardEventProperties.METHOD_FIELD_NAME, method,
                BardEventProperties.PATH_FIELD_NAME, path));
    addToPropertiesIfPresentInHeader(
        request, properties, "X-Transaction-Id", BardEventProperties.TRANSACTION_ID_FIELD_NAME);
```